src/components/Switch/Switch.component.tsx

```diff
@@ -22,25 +22,30 @@ import {StyledSwitch} from './Switch.styles';
  *
  */
 export const Switch = (props: SwitchProps) => {
-    const {id, label, subLabel, error, className = '', fullWidth = false, ...restProps} = props;
+    const {id, label, subLabel, error, className = '', fullWidth = false, disabled, size = 'md', ...restProps} = props;
```

Comment:
Is there any need for constants for md sm etc. One use case could be that we can give detailed name so for others it would be easier to understand these sizes and not have to find the comment in the switch file.